src/app/lib/config/services/index.jsx

```diff
@@ -18,6 +18,15 @@ export default {
     imageCopyrightOffscreenText: 'Default image copyright text, ',
     service: 'default',
     themeColor: '#000',
+    svg: {
```

Comment:
Hmm, not sure of this, but also not sure what to put in its place

Reply:
Mayb we can put an empty group `<g/>`

Reply:
I would rather make SVGs optional in psammead-brand and not include one

Reply:
I would almost be tempted to leave it out of the defaults, they really need to be redone and nothing uses them/falls back to them. 

@sareh @pjlee11 @jblaut what do you think?

Also

Ive raised https://github.com/bbc/psammead/issues/472 to catch an error I found if the svg isnt provided

Reply:
What is the SVG we are suggesting inside of the `<g>`? I think without UX input we probably shouldn't define a default SVG, happy to leave it out and open a new issue so we don't hold up this PR